controllers/apply/under10k/confirmation.js

```diff
@@ -42,46 +42,59 @@ module.exports = function ({ locale, data = {} }) {
     );
 
     function enConfirmationBody() {
-        return `<h2>We’ve just sent an email to your main and senior contact</h2>
-<p>This is just a confirmation email, with a summary of your answers (in case you want to look back at them at any point).</p>
-
-<h2>Now we’ve got your application – we'll start assessing it as soon as we can</h2>
-<p>We’ll look at your idea and do some checks. <strong>It’ll take us around ${getLeadTimeWeeks(
-            country
-        )} weeks to decide to fund your project or not</strong>.</p>
-
-<h2>While we’re assessing your application – we might get in touch</h2>
-<p>We don’t always do this. It’s only if we need a bit more information. So don’t worry if you don’t hear from us.</p>
-
-<h2>We’ll let <span data-hj-suppress>${mainContact.fullName} and ${
-            seniorContact.fullName
-        }</span> know our decision by email</h2>
-<p>We’ll email <strong data-hj-suppress>${
-            mainContact.email
-        }</strong> and <strong data-hj-suppress>${
-            seniorContact.email
-        }</strong>. Make sure to check the junk mail too (just in case).</p>
-
-<h2>If you have any questions, you can contact us in the meantime</h2>
-<p>Please call ${getPhoneFor(country)} or email <a href="mailto:${getEmailFor(
-            country
-        )}">${getEmailFor(country)}</a> and we will be happy to help you.</p>
-
-<p>
-    Good luck with your application!<br />
-    <strong>The National Lottery Community Fund</strong>
-</p>
-        `;
+        return stripIndents`
```

Comment:
Apologies for the diff here, was finding the template hard to parse. Used `stripIndents` to allow me to tidy it up.